### ISSUE TITLE
Need more proofreading & explanation

### ISSUE THREAD

**Issue by k2lh** [carbon-design-system/carbon#6372]
In the main carbondesignsystem site, the Dashboard section in the Data Viz chapter (section?), there's this text:

```
Use consistent color assignments
Always use consistent colors for **the each** data set within a dashboard.
```

1. Take out either 'the' or 'each', guys. 
2. This is... ambiguous, at best. I've had data scientists interpret this as "use consistent colors for _each_ data viz" and others interpret it as using the same color set across all data viz. If a dashboard has four different visualizations, a separate set of colors for each data viz means oodles of colors that can really impact accessibility and usability. 

Just throwing that out there, for the team to come up with a better way to describe your recommendations on how colors should be used. Strongly suggest some visuals that show what a good dashboard looks like vs a bad one (think of the Material doc style that shows "do this" and "don't do this").
